### ISSUE TITLE
gui : ssh-plugin code is hidden

### ISSUE THREAD

**Issue by toxic0berliner** [cronicle-edge/cronicle-edge#118]
*Steps to reproduce*:
Using a mobile browser (FF or Chrome on Android) 
Open any job for edit that uses the ssh plugin 
Type or look at a long command line 

*Current Behavior*: 
The rightmost part of the command is hidden and invisible therefore uneditable, even loving the edit cursor by sliding a finger to the right on the spacebar

*Desired behavior*
Wrap the lines as there is line numbering on the left, allowing to avoid horizontal scrolling entirely.
Other option would be to manage horizontal scrolling and adding something to keep the editing cursor always visible.

**Comment by mikeTWC1984** [comment]
Yeah, code editor doesn't look right on mobile devices indeed. It wasn't really optimized for that though, I did some effort to make it look more or less OK on 14" laptop screens but didn't get for a smaller device yet. I'll see what can be done
